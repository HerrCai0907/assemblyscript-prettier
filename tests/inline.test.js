import * as prettier from "prettier";
import asPlugin from "../src/plugin.js";

test("inline variable", async () => {
  const originCode = `
  
  @inline
  const EXPECT_MAX_INDEX = 2147483647;
  
      `;

  const formattedCode = await prettier.format(originCode, {
    parser: "typescript",
    plugins: [asPlugin],
  });

  expect(formattedCode).toMatchSnapshot();
});
