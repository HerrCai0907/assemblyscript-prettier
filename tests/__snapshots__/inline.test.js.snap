// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`inline variable 1`] = `
"
@inline
const EXPECT_MAX_INDEX = 2147483647;
"
`;
